packages/api/tests/lib/test-reconciliationReport.js

```diff
@@ -246,6 +246,7 @@ test('compareEsGranuleAndApiGranule returns true for matching granules', (t) =>
     granuleId: cryptoRandomString({ length: 5 }),
   };
   const granule2 = { ...granule };
+  granule2.files = [];
```

Comment:
🐜 
I think you could also write it this way:
```suggestion
const granule2 = { ...granule, files: [] };
```